interview-questions.md

```diff
@@ -7,62 +7,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: I know if/else are conditional statements, which applys to a condition and if true, theres a command to follow. If not true theres also a command to follow. 
 
-  Researched answer:
+  Researched answer:A way for computers to make decisions based on conditions.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: git is like the control, and github is the like the cloud. 
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them.
```

Comment:
Researched answers CANNOT be copied from the internet.  This must be formed in your own words.